linera-core/src/client/mod.rs

```diff
@@ -1356,14 +1374,13 @@ where
 
     /// Processes the result of [`synchronize_received_certificates_from_validator`] and updates
     /// the tracker for this validator.
-    #[tracing::instrument(level = "trace", skip(tracker, certificates))]
+    #[tracing::instrument(level = "trace", skip(received_certificates))]
     async fn receive_certificates_from_validator(
         &self,
-        name: ValidatorName,
-        tracker: u64,
-        certificates: Vec<Certificate>,
+        received_certificates: ReceivedCertificatesFromValidator,
```

Comment:
We can do this trick here where we destructure `ReceivedVertificatesFromValidator` into its elements.